### PR TITLE
Allow PHP8

### PR DESCRIPTION
**Pull request by T2L**
| Q             | A
| ------------- | ---
| Branch        | master for features and deprecations
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | ?
| Deprecations? | ?
| Documented?   | no
| Fixed tickets | #843 

This PR adds support of PHP8 which going to be released 26/11/2020

### CODE REVIEW

**Comment by T2L** [comment]
So, seems like this PR makes the project PHP 8 compatible.

However, I think it's worth to wait for https://github.com/paratestphp/paratest/pull/559 to be committed as well as the new release (and avoid dev-master dependency)

**Comment by veewee** [comment]
Looks nice, thanks!
We'll wait for paratest or until PHP 8 gets released - whichever comes first :)

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": ">=7.3",
```
Comment (veewee) [inline composer.json:17]:
```suggestion
        "php": "^7.3 || ^8.0",
```

**Comment by T2L** [comment]
Thanks :)

Btw, there is a new release of https://github.com/paratestphp/paratest/releases/tag/v6.0.0

**Comment by veewee** [comment]
Awesome! Checked this morning and it wasnt there yet. Will release PHP8 compatible version today as well ;)

**Pull request merged**